src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -268,6 +277,22 @@ public static void start(Method testMethod) {
   }
 
   private void stopLogging() {
+    NetworkTrafficInterceptor networkTrafficInterceptor = driver.getProxy();
+    if ( networkTrafficInterceptor != null ) {
+      Har har = networkTrafficInterceptor.getHar();
+      for (HarEntry entry : har.getLog().getEntries()) {
+        URL url = null;
+        try {
+          url = new URL(entry.getRequest().getUrl());
+          if (url.getHost().contains("wikia")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS134)